test/uk/gov/hmrc/gform/sharedmodel/formtemplate/generators/DestinationGen.scala

```diff
@@ -28,7 +28,12 @@ trait DestinationGen {
       r <- Gen.alphaNumStr
     } yield DestinationId(f.toString + r)
 
-  def dmsFormIdGen: Gen[String] = PrimitiveGen.nonEmptyAlphaNumStrGen
+  def dmsFormIdGen: Gen[String] = PrimitiveGen.nonEmptyAlphaNumStrGen.map { dmsFormId =>
+    if (dmsFormId.length > 12)
```

Comment:
I dont think we need this check. Just replace with `dmsFormId.take(12)`

Reply:
Yes. Changed.